### ISSUE TITLE
Remember the last used folder for materials and others

### ISSUE THREAD

**Issue by rogeriodec** [godotengine/godot#27474]
**Godot version:**
3.1

**OS/device including version:**
Windows 10 x64 / NVIDIA GeForce GTX 1060 6GB / Intel Core i7 4790K

**Issue description:**
If I do repetitive tasks like insert materials that are in subfolders, Godot always returns to the `res://` folder, so for each operation, I have to repeat all steps until finding that folder again.

**Steps to reproduce:**
1) Load a material (which is located on a subfolder) to a mesh 
2) Repeat the process to other mesh and you'll have to repeat all steps to find the subfolder.

**Comment by asheraryam** [comment]
I agree that it should remember folders for different file types, that would be very nice.

However note that on the bottom left there's a "Recent" list and the last folder you visited is usually there at the top, so you don't have to press many folders, just pick from the Recents list.

**Comment by rogeriodec** [comment]
> I agree that it should remember folders for different file types, that would be very nice.
> 
> However note that on the bottom left there's a "Recent" list and the last folder you visited is usually there at the top, so you don't have to press many folders, just pick from the Recents list.

True. I have not noted this. Thanks!
